tests/test_fork_provider.py

```diff
@@ -156,3 +156,76 @@ def test_get_receipt(connected_mainnet_fork_provider, fork_contract_instance, ow
     assert receipt.txn_hash == actual.txn_hash
     assert actual.receiver == fork_contract_instance.address
     assert actual.sender == receipt.sender
+
+
+@pytest.mark.sync
```

Comment:
Just an FYI - these fork tests currently don't run in CI.

I am wondering if we actually need the `@pytest.mark.sync` for these tests.

If it not actually making provider RPC requests, we maybe can drop this.

Reply:
I'm not sure what that is and couldn't immediately find that in pytest docs or conftest.  Will leave this to you.